import_and_export_data/formats/product_model.rst

```diff
@@ -0,0 +1,28 @@
+Product model data structure
+============================
+
+Product models are exported in a file with the following fields:
```

Comment:
finish with a .

Reply:
All the format have the same sentence, finished with ':' so I prefer standardization

Reply:
Well it's a documentation, and it should be english ^^. I agree with @pierallard it should a '.' to end the sentence.

Reply:
Fix for product & product model.